pkg/cedar/intake/schemas/easi_system_intake.json

```diff
@@ -54,152 +54,336 @@
       ],
       "properties": {
         "adminLead": {
-          "type": "string"
+          "type": "string",
+          "description": "Government Admin responsible for handling request",
+          "examples": [
+            "John Doe"
+          ]
         },
         "archivedAt": {
-          "type": "string"
+          "type": "string",
+          "description": "Timestamp of when request was archived",
+          "examples": [
+            "2022-02-17 14:34:43.405934+00"
+          ]
         },
         "businessNeed": {
-          "type": "string"
+          "type": "string",
+          "description": "Business Need (problem space",
+          "examples": [
+            ""
```

Comment:
Did you have an example in mind for this?

Reply:
I had a description it just wasn't getting populated due to the non-escaped commas in the description, fixed.